javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,120 @@
+/* My age in 2027*/ 
+let yearOfBerth;
+let yearFuture;
+yearOfBerth = 1985; // integer, 
+yearFuture = 2027; // integer, 
+const age = yearFuture - yearOfBerth;
+console.log('You will be' + age+ 'years old in' + yearFuture);
+
+/* Dog age*/
+let dogYearOfBirth;
+let dogYearFuture;
+dogYearOfBirth = 2017;
+dogYearFuture = 2027;
+const dogYear = dogYearFuture - dogYearOfBirth;
+let shouldShowResultInDogYears = false;
+if (shouldShowResultInDogYears === true)
+{
+	console.log('Your dog will be' + dogYear + ' human years old' + dogYearFuture );
+}
+else {
+	console.log('Your dog will be' + dogYear*7 + 'dog years old' + dogYearFuture);
+}
+
+/* House */
+// Formula housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+//Your friend Peter is considering a house that is 8m wide, 10m deep and 10m high. The garden size is 100m2. The house costs 2.500.000
+
+// find out the value of high, wide and deep assign the value to  volumeInMeters
+// Then put the values into the formula
+var volumeInMetersForPeter = 8 * 10 * 10;
```

Comment:
Generally, in newer JS it's a better practice to avoid the var variables. Instead you should try using let and const depending on if you intend to change the variable or not.